script/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "binanceAssetsUrl": "https://explorer.binance.org/api/v1/assets?page=1&rows=1000"
```

Comment:
`const binanceAssetsUrl = config.getConfig("binanceAssetsUrl", "https://explorer.binance.org/api/v1/assets?page=1&rows=1000");`

this does not make sense

Reply:
This should be statically typed